syntax = "proto3";

package Global.Protocol.User;

message RequestLogin
{
	string account = 1;
	string password = 2;
	int32 role = 3;
	string secret_key = 4;
}

message ResponseLogin
{
	int32 err_code = 1;
	string err_string = 2;
	int64 user_id = 3;
}